packages/react-server-dom-webpack/src/__tests__/ReactFlightDOMBrowser-test.js

```diff
@@ -525,6 +525,53 @@ describe('ReactFlightDOMBrowser', () => {
     expect(container.innerHTML).toBe('{"foo":1}{"foo":1}');
   });
 
+  it('should handle deduped props of elements in fragments', async () => {
+    let resolveFooClientComponentChunk;
+
+    const FooClient = clientExports(
+      function Foo({children, item}) {
+        return children;
+      },
+      '1',
+      '/foo.js',
+      new Promise(resolve => (resolveFooClientComponentChunk = resolve)),
+    );
+
+    const shared = <div />;
+
+    function Server() {
+      return (
+        <FooClient track={shared}>
+          <React.Fragment>{shared}</React.Fragment>
```

Comment:
We don't hit this particular case without the Fragment though the Fragment isn't relevant. It just helps us trigger the case where deduped props of elements are resolved.